examples/volumes/flexvolume/deploy/deploy.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+
+# TODO change to your desired driver.
+VENDOR=${VENDOR:-k8s}
+DRIVER=${DRIVER:-dummy}
```

Comment:
Hmm.. shall we fail if these variables are not set? How about we add a check and exit the container with a log? These variables have to be defined.

Reply:
I'd like to make it minimal effort for users to get a basic example running. Currently the user only has to copy the driver into the appropriate directory, then build and deploy.

I'll add a comment about that in the README

Reply:
ACK